src/test/test-condition.c

```diff
@@ -37,66 +37,82 @@ static void test_condition_test_path(void) {
         Condition *condition;
 
         condition = condition_new(CONDITION_PATH_EXISTS, "/bin/sh", false, false);
+        assert_se(condition);
```

Comment:
The first thing in `condition_test` is `assert(condition)`. Adding this redundant test doesn't hurt of course, but I don't think it adds any value either.